dapps/marketplace/src/hoc/withMessagingStatus.js

```diff
@@ -5,9 +5,9 @@ import query from 'queries/WalletStatus'
 
 import get from 'lodash/get'
 
-function withMessagingStatus(WrappedComponent) {
+function withMessagingStatus(WrappedComponent, { excludeData } = {}) {
```

Comment:
Maybe only 1 comment. Is there any substantial effect to `excludeData`? I mean it is still being queried, the only difference is that is not being passed down to the wrapped component. Is the idea that this conserves memory?

Reply:
Added that as `messagingStatus` prop causes some components to rerender after a few milliseconds. This sometimes results in failing marketplace tests. But it won't be necessary once Nick's #3255 PR gets merged.

Reply:
Ahh haven't thought of that! Makes sense, thanks for explaining it.